<launch>
  <!-- <node pkg="tf" type="static_transform_publisher" name="odom_transform" args="0 0 0 0 0 0 /map /odom 20"/> -->
  <node pkg="fake_localization" type="fake_localization" name="fake_localization" respawn="true" output="screen"/>

  <!-- <node name="map_server" pkg="map_server" type="map_server" args="$(find stage_nav)/maps/laas_adream_stage.yaml"/> -->

  <node pkg="global_planner" type="planner" name="gbl_planner" output="screen">
    <rosparam file="$(find stage_nav)/params/global_costmap_params.yaml" command="load" ns="costmap"/>
  </node>

</launch>
